src/browser/queries/types.ts

```diff
@@ -0,0 +1,69 @@
+import {
+    Config as BaseConfig,
+    BoundFunction as BoundFunctionBase,
+    queries,
+    waitForOptions,
+    SelectorMatcherOptions,
+    MatcherOptions,
+} from "@testing-library/dom";
+import { SelectorsBase } from "./wdio-types.js";
+
+export type Queries = typeof queries;
+export type QueryName = keyof Queries;
+
+export type Config = Pick<
+    BaseConfig,
+    | "asyncUtilTimeout"
+    | "computedStyleSupportsPseudoElements"
+    | "defaultHidden"
+    | "testIdAttribute"
+    | "throwSuggestions"
+>;
+
+export type WebdriverIOQueryReturnType<Element, ElementArray, T> = T extends Promise<HTMLElement>
```

Comment:
I don't think it's a good idea to use webdriverio naming all around. We are planning to begin distancing testplane from wdio

Reply:
renamed to testplane